codegen/Cargo.toml

```diff
@@ -22,6 +22,7 @@ syntex = { version = "0.30.0", optional = true }
 
 [dependencies]
 Inflector = "0.2.0"
+lazy_static = "0.1"
```

Comment:
For better or worse I'm a big fan of specifying as much of the version as possible.  Any complaints for locking this to 0.1.16?